frontend/src/bundles/studio/components/timeline/timeline.tsx

```diff
@@ -49,18 +33,32 @@ const Timeline: React.FC<Properties> = ({
 
             const updatedSpan = activeItem.getSpanFromResizeEvent?.(event);
 
-            if (!updatedSpan || activeItemType === RowNames.SCRIPT) {
```

Comment:
Please, leave this condition and remove `resizeScript` reducer. Scripts must not be resizable.

Reply:
Done)